README.md

```diff
@@ -85,6 +85,34 @@ func withoutContext() {
 }
 ```
 
+Global Logger
+--------
+The `americanas-go/log` provides top level logging function, however by default they do nothing (NoOp). You can define your global logger, after you instantiate the desired implementation, by using the `log.SetGlobalLogger`.
+
+```go
+package main
+
+import (
+	"context"
+
+	"github.com/americanas-go/log"
+
+	//"github.com/americanas-go/log/contrib/go.uber.org/zap.v1"
+	//"github.com/americanas-go/log/contrib/rs/zerolog.v1"
+	"github.com/americanas-go/log/contrib/sirupsen/logrus.v1"
+)
+
+func main() {
+	//logger := zap.NewLogger()
+	//logger := zerolog.NewLogger()
+	logger := logrus.NewLogger()
+	log.SetGlobalLogger(logger)
```

Comment:
This call (log.Set...) could be added to NewLogger functions so you dont have do remember to call it every time you instantiate a Logger... what you think?

Reply:
Actually, this already happens, the `log.SetGlobalLogger` is called on the NewLogger functions. As I explained in the PR description, this is a code smell to have side effects inside a constructor/factory. I didn't remove this behavior because it would be a breaking change. Hence, I made the documentation this way in order to promote a better practice to have side effects done in an isolated and explicit way, paving the path to remove this on a v2.